osconfig_tests/main.go

```diff
@@ -40,7 +39,7 @@ var (
 
 var testFunctions = []func(context.Context, *sync.WaitGroup, chan *junitxml.TestSuite, *log.Logger, *regexp.Regexp, *regexp.Regexp){
 	packagemanagement.TestSuite,
-	inventory.TestSuite,
+	// inventory.TestSuite,
```

Comment:
debug edit?

Reply:
oh yeah! just to save some time, i removed it, since test-suite-filter didn't work. will remove it.